Eradication/VectorSpeciesParameters.h

```diff
@@ -89,9 +93,6 @@ namespace Kernel
         VectorGeneDriverCollection gene_drivers;
 
     protected:
-        friend class VectorSpeciesCollection;
-
-        VectorSpeciesParameters( int _index );
```

Comment:
The reason I had the constructor projected and had VectorSpeciesCollection as a friend was so that only VectorSpeciesCollection could create instances of VectorSpeciesParameters.  Did you have a reason for changing it?